positional/src/parsed_field.rs

```diff
@@ -22,14 +22,20 @@ impl<'s> PositionalParsedField<'s> {
 
     /// output a string representation of the parsed value
     /// trimming is done by the library based on the declared positional row configurations
-    pub fn to_value(&self) -> &'s str {
-        let slice_start = self.offset;
-        let slice_end = self.offset + self.size;
-        let raw_value = &self.row[slice_start..slice_end];
+    pub fn to_value(&self) -> String {
+        // we don't take into consideration the empty string (or generally an out of bound string)
+        // because we are already checking for string correctness in the main `from_positional_row`
+        // function
+        let raw_value = self
+            .row
+            .chars()
```

Comment:
I think that we could add that library as a dependency behind a feature!

Reply:
> This is, however, a big improvement over the existing code since it doesn't rely on byte boundaries, but actually works on unicode points

I think this is wrong, since string indexing in rust already works on unicode points

Reply:
It most definitely does not: https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=63b9782f5832a970b80c0d724bb39787

Reply:
> I think that we could add that library as a dependency behind a feature!

I'm not sure. Gating additional content behind feature gates makes sense, I'm not sure the same can be said about changing existing behaviour. At any rate, it's out of scope for this PR, so let's discuss that elsewhere :)

Reply:
Hang on, I got confused earlier. My original comment was right, and the playground example is fine. 
It compares `s.get(0..1)` with `s.chars().take(1)`. In both cases, you'd expect to get one character, but the get version returns None because it's not on a valid UTF-8 boundary. I stand by my original comment.

Reply:
you're right, I was really sure about this... Oh well, thanks for teaching me something

Reply:
so it's better to use the chars iterator right?

Reply:
Yes